package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hypercerts-org/marketplace-sdk",
-  "version": "0.5.3",
+  "version": "0.5.4",
```

Comment:
i just merged the other PR so this needs updating

Reply:
What's the correct version number to enter here?